types/reactstrap/lib/FormText.d.ts

```diff
@@ -2,7 +2,7 @@ import { CSSModule } from '../index';
 
 interface Props {
   inline?: boolean;
-  tag?: string;
```

Comment:
Based on [this line](https://github.com/reactstrap/reactstrap/blob/master/src/FormText.js#L9), I expect this should be `string | React.ReactType`. Am I incorrect?

Reply:
@weswigham The change to FormText as I've made it is consistent with other elements. Please see the reactstrap code and corresponding type for CardText below:

https://github.com/reactstrap/reactstrap/blob/master/src/CardText.js#L7
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/reactstrap/lib/CardText.d.ts#L4